.github/workflows/docker.yaml

```diff
@@ -322,7 +322,8 @@ jobs:
           fi
 
           echo '::group::docker compose up'
-          timeout 5m docker compose --file "$FILE" up --wait
+          docker compose --file "$FILE" build
+          timeout 3m docker compose --file "$FILE" up --wait
```

Comment:
We should make the timeout configurable while we're editing this file.

You'll also need to `git tag -f v1 && git push -f origin v1` for GitHub runners to pick up this change.

Reply:
I don't see any docs on the type supported by workflow. I did find this blog post:  
https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/

I'll make this a string for now.